registry/1000g-subset.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "1000g-subset",
+    "description": "Subset of 1000G Low Coverage WGS Data",
+    "target": "1000g-subset/reads",
```

Comment:
maybe, `1kg-subset/reads` or `1kg/<insert_description>`

Actually, in the spec doc, we should talk about optimal "directory" structure for the S3 data.